jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/JibPluginConfigurationTest.java

```diff
@@ -22,47 +22,43 @@
 import com.google.common.collect.ImmutableSet;
 import java.nio.file.Paths;
 import java.util.List;
+import java.util.Properties;
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.project.MavenProject;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
 
 /** Tests for {@link JibPluginConfiguration}. */
+@RunWith(MockitoJUnitRunner.class)
 public class JibPluginConfigurationTest {
 
+  private static final MavenProject project = new MavenProject();
+  @Mock private MavenSession session;
+  private Properties sessionProperties = new Properties();
   private JibPluginConfiguration testPluginConfiguration;
 
-  private static void clearProperties() {
-    System.clearProperty("jib.from.image");
-    System.clearProperty("jib.from.credHelper");
-    System.clearProperty("image");
-    System.clearProperty("jib.to.image");
-    System.clearProperty("jib.to.tags");
-    System.clearProperty("jib.to.credHelper");
-    System.clearProperty("jib.container.appRoot");
-    System.clearProperty("jib.container.args");
-    System.clearProperty("jib.container.entrypoint");
-    System.clearProperty("jib.container.environment");
-    System.clearProperty("jib.container.format");
-    System.clearProperty("jib.container.jvmFlags");
-    System.clearProperty("jib.container.labels");
-    System.clearProperty("jib.container.mainClass");
-    System.clearProperty("jib.container.ports");
-    System.clearProperty("jib.container.useCurrentTimestamp");
-    System.clearProperty("jib.container.user");
-    System.clearProperty("jib.container.workingDirectory");
-    System.clearProperty("jib.extraDirectory.path");
-    System.clearProperty("jib.extraDirectory.permissions");
+  private void clearProperties() {
```

Comment:
I don't think we need this method anymore? `project` and `sessionProperties` should be new on every test that is run in this class.